src/notifications/mod.rs

```diff
@@ -25,15 +25,27 @@ pub struct NotificationMessage {
   timestamp: String,
 }
 
-fn fetch_webhook_url() -> Result<String, VarError> {
-  env::var(WEBHOOK_URL)
+fn fetch_webhook_url() -> String {
+  fetch_var(WEBHOOK_URL, "")
+    .trim_start_matches('"')
+    .trim_end_matches('"')
```

Comment:
We shouldn't need this anymore as long as the env var is passed in correctly

Reply:
Im going to keep it for now, just in case.